ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,106 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+<li> CMake 3.13 or higher
+<li> G++ 8.4 or higher <p>
```

Comment:
The formatting here is a little weird, we should use Markdown syntax rather than HTML. Also, do we have a minimum Clang version?

Reply:
It seems to work on clang 6.0.0 so im not too sure if there is a minimum version required